Gems/AtomLyIntegration/CommonFeatures/Code/Source/CoreLights/AreaLightComponentController.cpp

```diff
@@ -601,6 +601,16 @@ namespace AZ::Render
         }
     }
 
+    AZ::Aabb AreaLightComponentController::GetLocalVisualizationBounds() const
+    {
+        if (m_lightShapeDelegate)
+        {
+            return m_lightShapeDelegate->GetLocalVisualizationBounds();
+        }
+
+        return AZ::Aabb::CreateCenterRadius(AZ::Vector3::CreateZero(), 0.5f);
```

Comment:
`CreateFromPoint` - main thing I try to avoid is thinking any specific size is "large" or "small" for everyone. In the unlikely event someone makes a game about ants using all real physical units, 0.5m might be way too big :).

Reply:
Excellent will change, and good point! 🙂